index.html

```diff
@@ -59,10 +58,269 @@ <h2 class="headline-text1">I’m Wahab.
+
+                            
+                            <p class="role role-text">BackEnd Dev</p>
+
+                            <img class="container" src="assets/Counter.png" alt="Counter">
+
+                            <p class="year-label year-text">2015</p>
+                        </div>
+                    </div>
+
+                    <p class="text-after-Frame2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                    <div class="tags">
+                        <div class="tag-html"><p class="tag-text-html">HTML</p></div>
+                        <div class="tag-css"><p class="tag-text-css">CSS</p></div>
+                        <div class="tag-js"><p class="tag-text-js">JavaScript</p></div>
+                    </div>
+
+                    <div class="action-container">
+                        <button class="normal-button">
+                            <p class="project-text">See Project</p>
+                        </button>
+                    </div>
+
+
+                </div>
+
+            </div>
+
+            <div class="Card-works">
+                <div class="snapshoot-container">
+                    <img src="assets/Snapshoot-Portfolio-3.png" alt="Snapshoot Portfolio-3">
+                </div>
+                <div class="left-block">
+                    <div class="primary-text">
+                        <h2 class="project-title">Multi-Post
+                            Stories</h2>
+                        <div class="frame-2">
+                            <div class="client">
+                                <p class="client-text">CANOPY</p>
+                            </div>
+
+                            <img class="container" src="assets/Counter.png" alt="Counter">
+
+                            
+                            <p class="role role-text">BackEnd Dev</p>
+
+                            <img class="container" src="assets/Counter.png" alt="Counter">
+
+                            <p class="year-label year-text">2015</p>
+                        </div>
+                    </div>
+
+                    <p class="text-after-Frame2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                    <div class="tags">
+                        <div class="tag-html"><p class="tag-text-html">HTML</p></div>
+                        <div class="tag-css"><p class="tag-text-css">CSS</p></div>
+                        <div class="tag-js"><p class="tag-text-js">JavaScript</p></div>
+                    </div>
+
+                    <div class="action-container">
+                        <button class="normal-button">
+                            <p class="project-text">See Project</p>
+                        </button>
+                    </div>
+
+
+                </div>
+
+            </div>
+
+            <div class="Card-works">
+                <div class="snapshoot-container">
```

Comment:
- [ ] Please check the original design as suggested in the below [video](https://youtu.be/jiIed23OPMA) recorded for goo understanding 😉